Shared/Samples/Apply scheduled updates to preplanned map area/README.md

```diff
@@ -0,0 +1,47 @@
+# Apply scheduled updates to preplanned map area
+
+Apply scheduled updates to a downloaded preplanned map area.
+
+![Image of Apply scheduled updates to preplanned map area sample](apply-scheduled-updates-to-preplanned-map-area.png)
+
+## Use case
+
+With scheduled updates, the author can update the features within the preplanned areas on the service once, and multiple end-users can request these updates to bring their local copies up to date with the most recent state. Importantly, any number of end-users can download the same set of cached updates which means that this workflow is extremely scalable for large operations where you need to minimize load on the server.
+
+This workflow can be used by survey workers operating in remote areas where network connectivity is not available. The workers could download mobile map packages to their individual devices and perform their work normally. Once they regain Internet connectivity, the mobile map packages can be updated to show any new features that have been added to the online service.
+
+## How to use the sample
+
+When the sample loads, it will display an offline map. Check for available updates, and show update availability and size. Tap the button to apply the updates to the local offline map and show the results.
```

Comment:
```suggestion
When the sample loads, it will display an offline map, check for available updates, and show update availability and size. Tap the button to apply the updates to the local offline map and show the results.
```

Is this supposed to be one sentence?